src/BlockTypes/Checkout.php

```diff
@@ -49,6 +50,21 @@ public function register_block_type() {
 	 * @return string Rendered block type output.
 	 */
 	public function render( $attributes = array(), $content = '' ) {
+		$this->enqueue_data( $attributes );
```

Comment:
why is this only happening in Checkout and not Cart?

Reply:
oversight :)

Reply:
Fixed in latest commits

Reply:
yeah it was breaking Cart in frontend